uiautomator-espresso/javascript/README.md

```diff
@@ -0,0 +1,110 @@
+# Javascript samples using UIAutomator/ Espresso framework
+
+## I. Setup environment on Mac
+
+#### Homebrew
+
+- [Homebrew](https://brew.sh/) is a package manager for the Mac.
+-  To install Homebrew, open terminal and type the following command:
+
+```bash
+ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+```
+
+- This will install Homebrew on your Mac. To check the version, type the following command:
+
+```bash
+brew -v
+```
+
+#### Node and npm
+
+- To install Node via Homebrew, type the following command:
+
+```bash
+brew install node
+```
+
+- To check if you have [Node.js](https://nodejs.org/en/) and [npm](https://www.npmjs.com/) installed, run this command in your terminal:
+
+```bash
+node -v
+npm -v
+```
+
+- To update your npm to latest, type this into in your terminal:
+
+```bash
+npm install npm@latest -g
+```
+
+## Install the uiautomator-espresso samples
```

Comment:
Install -> Clone

Reply:
Keep install, with javascript we need to install the dependency (request).